src/initialize-node.ts

```diff
@@ -20,7 +20,7 @@ import pkgDir from 'pkg-dir';
 import { Product } from './common/Product.js';
 
 export const initializePuppeteerNode = (packageName: string): PuppeteerNode => {
-  const puppeteerRootDirectory = pkgDir.sync(__dirname);
+  const puppeteerRootDirectory = pkgDir.sync(process.cwd());
```

Comment:
This should resolve to `node_modules/puppeteer` when the user installs it. Could you try the following?

```
import path from 'path';
import { fileURLToPath } from 'url';
path.dirname(fileURLToPath(import.meta.url)
```